qemu/tests/cfg/x86_cpu_flags.cfg

```diff
@@ -48,6 +48,21 @@
                             expect_items = 48
                             cpu_model_flags = ",-la57"
                             no_flags = "la57"
+                - intel_pt:
+                    type = x86_cpu_flag_intel_pt
+                    flags = "intel_pt"
+                    required_qemu = [4.2.0, )
+                    set_pt_mode = "rmmod kvm_intel && modprobe kvm_intel pt_mode=%s"
+                    get_pt_mode = "cat /sys/module/kvm_intel/parameters/pt_mode"
+                    variants:
+                        - @default_model:
+                            auto_cpu_model = no
+                            cpu_model = host
+                        - host_model:
```

Comment:
These  two variants seem conversed to me. The first one should be host_model.

Reply:
update and thanks.